### DISCUSSION TITLE
auto-fill

### DISCUSSION THREAD

**Discussion by aloispichler** [syl20bnr/spacemacs#16248] [Q&A] [answered]
I could not find out how to disable `auto-fill` in `.spacemacs`. I have tried `(auto-fill-mode -1)`, `(add-hook 'lateX-mode-hook 'turn-off-auto-fill)` and removed all hooks, but nothing worked. Any help welcome!

**Comment by alexey0308** [comment, marked as answer]
Hi, please take a look at [the documentation](https://develop.spacemacs.org/layers/+lang/latex/README.html#auto-fill) of the latex-mode

```
(latex :variables latex-enable-auto-fill nil))
```

does it solve your issue?

**Reply by aloispichler** [reply]
This seems to solve the problem. I have not seen this option before, thank you!
